README.md

```diff
@@ -210,6 +210,18 @@ services:
       - { name: kernel.event_listener, event: kernel.terminate }
 ```
 
+## HTTP Client
```

Comment:
WDYT to implement a [client scope](https://symfony.com/doc/current/components/http_client.html#configuration)? In this way we can inject scoped client

Reply:
Hi @dirk39 - I really like the idea but I struggled to find a way to pass a
custom class as a scoped client, I believe only options are passed. Do you
know how to do this?

Reply:
You can declare a service like `zipkin.http_client` that implements the `ZipkinBundle\Components\HttpClient` and inside the `services.yaml` you can say that any argument that implements `HttpClientInterface` named `$zipkinHttp` will use the `zipkin.http_client`:

```
Symfony\Contracts\HttpClient\HttpClientInterface $zipkinHttp: '@zipkin.http_client'
```

Reply:
I like the idea of having a `zipkin.http_client`.

Regarding 

> inside the services.yaml you can say that any argument that implements HttpClientInterface named $zipkinHttp will use the zipkin.http_client:

I think having an instrumented client should be transparent for users, hence asking users to change their classes to accept `zipkinHttp` over `httpClient` is leaking some implementation details and it is probably not necessary if we can override the `http_client` declaration. Does that make sense?

Reply:
In my idea, we leave the possibility to decide whether to use the Zipkin implementation or the default one. 
Effectively we can implement the `zipkin.http_client` and suggest in `README.md` how to set Zipkin client as an argument using argument override in `services.yaml`.
WDYT?